data/102/1021/en_US.json

```diff
@@ -5,7 +5,7 @@
 			"message": "Use of this feature requires a linked\nNintendo Network ID.\n\nYou can link an ID from User Settings.",
 			"short_description": "Unknown cause.",
 			"long_description": "Unknown cause.",
-			"short_solution": "Unknown solution.",
+			"short_solution": "Link a Nintendo/Pretendo Network ID to your user.",
```

Comment:
This goes for all the errors which you have done this with. If you are going to add a short solution, we ask that you also add a long solution. The long solution is what is going to be shown on the website, and supports markdown rendering